data/naltok jobs.txt

```diff
@@ -177,11 +177,10 @@ mission "Xochira Logistics Small Rush"
 		payment 5000 250
 		dialog phrase "generic cargo delivery payment"
 
-#todo: more variants. Also, figure out a way to sort out rep changes, and only make the Xochira hostile on boarding or destroying rather than when deadline is missed.
+# todo: more variants. Also, figure out a way to sort out rep changes, and only make the Xochira hostile on boarding or destroying rather than when deadline is missed.
```

Comment:
I actually got halfway through setting this up with governments, but got sidetracked before making the changes to the mission.
Boarding, capturing or destroying ships of the Xochira (training) government has the same effect on the Xochira reputation as doing it to Xochira ships (disabling them as you were asked to has no effect), so the reputation changes you've added won't be necessary.
The dialogs are a sensible addition though, though I'd like to rewrite them as a message from the Xochira rather than an out-of-universe message once I've got the Naltok speech pattern down.
I do want to keep the deadline though. If the reputation penalty and the dialog are removed from the on fail as you've done, I could probably keep it without causing issues.